motionplan/tpSpaceRRT.go

```diff
@@ -306,6 +306,36 @@ func (mp *tpSpaceRRTMotionPlanner) rrtBackgroundRunner(
 			rrt.solutionChan <- &rrtSolution{err: fmt.Errorf("TP Space RRT timeout %w", ctx.Err()), maps: rrt.maps}
 			return
 		}
+
+		// Check to see if the startPose is already within some predefined delta of the goalPose
+		if iter == 0 {
```

Comment:
We should probably actually move this up to where we iterate the goal map on line 224 and check for every goal node.